### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by mlocati** [oyejorge/less.php#329]
Hi @oyejorge 

Pull requests and issues haven't received any feedback from you since April... Is this project still supported?

**Comment by ivanjaros** [comment]
I guess everybody has moved to nodejs :(

**Comment by shtse8** [comment]
I am using this project. Hope it can keep supporting this.
Anyone fork this project to keep maintaining?

**Comment by ivanjaros** [comment]
Maybe you can try the leafo scss but that one is broken as well since june, but maybe it will work for your code.

**Comment by Asenar** [comment]
Hi @shtse8 , I forked it some days ago and rebase + merge the main pending pull request I saw.

**Comment by shtse8** [comment]
@Asenar Good job. I should use yours.

**Comment by shtse8** [comment]
@Asenar what is the difference between kohana-less and less.php?

**Comment by Asenar** [comment]
kohana-less is a module made for kohana (a framework I use). It is a wrapper around less.php to simplify its integration in Kohana3

less.php is the library from @oyejorge , I took the works of 5-6 people who made pull requests here, that i noted in the PR #338 ). I did that for my personal use (specially the `$parser->getVariables` method I completed), and shared it here.

I just noticed the travis part didn't worked anymore so I m investigating a bit…

**Comment by shtse8** [comment]
Hop your project can support composer, so I can directly use yours instead of this one without anyone maintaining.

**Comment by Asenar** [comment]
Yes it could, I almost did it but at the final step in packagist.org I found @concrete5 (with @concrete5-forks ) created a package for composer too 4 days ago.

So maybe its not so useful to have 2 different packages for the same thing…

@concrete5, do you plan to maintain it ?

**Comment by KorvinSzanto** [comment]
@Asenar we have no plans to actually maintain this package, we just published our fork so that we could stop using a custom repo. We needed https://github.com/oyejorge/less.php/pull/324 in order to continue using less.php.

**Comment by Asenar** [comment]
Ok, thanks. I added it too :)

I took theses PRs : #324 #325 #327 #328 #311 #258 #267 #263

**Comment by oyejorge** [comment]
As you've all noticed, I'm not able to maintain this project. I'm happy to add maintainers to this repo or link to a new one.

**Comment by Asenar** [comment]
Hi @oyejorge , I m a little busy right now but I started to take some PRs here to add them in my fork.

You can link it to https://github.com/Asenar/less.php :)

**Comment by MaxSem** [comment]
We at Wikimedia have [forked](https://github.com/wikimedia/less.php) this project and fixed issues with new PHP. We're open to pull requests!

**Comment by holtkamp** [comment]
@oyejorge maybe you can add @Asenar and @MaxSem and maybe other contributors as maintainers so that the amount of forks can be reduced?

**Comment by Asenar** [comment]
Hi,

Personally I moved to nodejs / webpack, and I slowly migrate my old projets that are still using less.php to it.

For them, I follow Wikimedia forks : https://github.com/wikimedia/less.php

**Comment by holtkamp** [comment]
Hi @Asenar thanks for pointing that out, will have a look and try to contribute to keep it going 😉
